packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -19,18 +19,16 @@ import {
   makeSlogCallbacks,
 } from './kernel-stats.js';
 
+import {
+  BeansPerMaxBlock,
```

Comment:
Assuming we do go with max-beans instead of max-computrons (see earlier comment), this might read better as `MaxBeansPerBlock`?